ilc/client/localStorageCache/LocalStorageCache.js

```diff
@@ -0,0 +1,24 @@
+export default class LocalStorageCache {
```

Comment:
I propose to remove wrapper end emulation and use library like https://www.npmjs.com/package/node-localstorage

Reply:
Updated